kratos/input_output/file_logger_output.cpp

```diff
@@ -0,0 +1,48 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Armin Geiser
+//
+//
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "input_output/file_logger_output.h"
+
+
+namespace Kratos
+{
+    namespace {
+    std::ofstream* CreateNewLogFile(const std::string& rFileName)
+    {
+        // The file stream created here on the heap is deleted in the destructor
+        // of the FileLoggerOutput
+        return new std::ofstream(rFileName);
```

Comment:
`new` is deprecated in c++20 and removed in c++23

why not using a unique_ptr?

Reply:
ah forget the first one, apparently I fell for an april fools joke :D

Reply:
I had to use new here, because the base class constructor requires a reference to an ostream. So i have to create the file stream in the initializer list of the base class, before i actually can store it as a member of the derived class...